gtdynamics/universal_robot/Joint.cpp

```diff
@@ -291,7 +282,7 @@ gtsam::Vector6_ Joint::poseConstraint(uint64_t t) const {
   Pose3_ wTc_hat = wTp * pTc;
 
   // Return the error in tangent space
-  return gtdynamics::logmap(wTc, wTc_hat);
+  return gtsam::logmap(wTc, wTc_hat);
```

Comment:
what are the changes in this file?  Are these yours or are they just some weird mistake in file diff?

Reply:
Edit: I see the note you made in the conversation now.